csharp/src/Microsoft.ML.OnnxRuntime/OnnxRuntime.shared.cs

```diff
@@ -48,16 +48,18 @@ public enum OrtLanguageProjection
     /// </summary>
     public sealed class OrtEnv : SafeHandle
     {
-        private static readonly Lazy<OrtEnv> _instance = new Lazy<OrtEnv>(()=> new OrtEnv());
+        private static Lazy<OrtEnv> _instance = null;
+        private LogLevel currentLogLevel;
 
         #region private methods
-        private OrtEnv()  //Problem: it is not possible to pass any option for a Singleton
+        private OrtEnv(LogLevel logLevel)
```

Comment:
I think the OrtEnv as a singleton doesn't allow any parameters to be specified as the comment also mentioned.

Reply:
I get hint from [here](https://stackoverflow.com/questions/4203634/singleton-with-parameters) that we can maintain the singleton and pass parameter via lazy initialization, and the parameter is used only when initializing the ortenv and can't be changed. 
The reason that some tests fail is [InferenceSession API ](https://github.com/microsoft/onnxruntime/blob/387ef0cdf1/csharp/src/Microsoft.ML.OnnxRuntime/InferenceSession.shared.cs#L793)initiated OrtEnv via OrtEnv.Handle, and it can be fixed by updating Handle() as well to be able to initiate the constuction of OrtEnv if instance is still null.  
I plan to contact with Dmitri to verify on that in the next step. Thanks!

Reply:
Yes, check with Dmitri